.scripts/update_refs.sh

```diff
@@ -55,7 +55,7 @@ if [[ "$CURRENT_BRANCH" == release-* ]]; then
 
   # Replace 0.0.0-dev refs with ${STACKABLE_RELEASE}.0
   # TODO (@NickLarsenNZ): handle patches later, and what about release-candidates?
-  SEARCH='stackable(0\.0\.0-dev|24\.7\.[0-9]+)' # TODO (@NickLarsenNZ): After https://github.com/stackabletech/stackable-cockpit/issues/310, only search for 0.0.0-dev
+  SEARCH='stackable(0\.0\.0-dev|24\.11\.[0-9]+)' # TODO (@NickLarsenNZ): After https://github.com/stackabletech/stackable-cockpit/issues/310, only search for 0.0.0-dev
```

Comment:
If all tags are using 0.0.0-dev (besides the hard-coded 24.3 stuff), then we can simplify this:

```suggestion
  SEARCH='stackable(0\.0\.0-dev)'
```